app/views/movies/index.html.erb

```diff
@@ -8,6 +8,10 @@
   <% if policy(Hall).index? %>
     <%= link_to "Check halls", halls_path, class: "btn btn-outline-secondary"%>
   <% end %>
+
+   <%# if policy(Discount).index? %>
```

Comment:
Generally it is not a good practice to add commented code. Often it stays forever as a comment.
In this case I assume that you will add some Discount policy soon and you will remember to change this comment into code, so I wont complain 😜
It looks good to me, approved 🏎️